### PR TITLE
force_use_tomcat_ipv4_and_optimize

### PR DESCRIPTION
**Pull request by cesarjorgemartinez**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,6 +15,8 @@ RUN yum update -y && yum install -y wget \
   && mvn package -Dmaven.test.skip=true \
   && mv target/perseo-core-*.war /usr/share/tomcat/webapps/perseo-core.war \
   && mkdir /var/log/perseo && chown tomcat:tomcat /var/log/perseo
+  # Force Tomcat to use ipv4 and optimize
+  && echo "JAVA_OPTS=\"-Djava.awt.headless=true -Xmx512m -XX:MaxPermSize=256m -XX:+UseConcMarkSweepGC -Djava.library.path=/usr/lib64:/usr/lib -Djava.security.egd=file:/dev/./urandom -Djava.net.preferIPv4Stack=true\"" >> /usr/share/tomcat/conf/tomcat.conf
```
Comment (fgalan) [inline Dockerfile]: As far as I understand, IPv6 is one of the features of the IoT Platform that we want to keep...

Ok with all the other optimizations but, do we really need to remove IPv6? Is there any impediment in the on premise context that needs so?

Comment (cesarjorgemartinez) [inline Dockerfile]: By now, we need here use ipv4, because perseo-fe try to connect with ipv4.
You can verify this inside perseo-fe?

Comment (fgalan) [inline Dockerfile]: Even in the case Perseo FE try to connect using always IPv6 (we'll check... @crbrox could you have a look?) maybe Tomcat could be configured to run in dual-stack? Is it not running in dual-stack by default?

Comment (cesarjorgemartinez) [inline Dockerfile]: Checked: I was failing! We need to correct this PR

**Comment by cesarjorgemartinez** [comment]
Enabled ipv4 and ipv6, we need to use both

**Comment by fgalan** [comment]
Closing on behalf of @cesarjorgemartinez

**Pull request closed without merging**